content/en/docs/tasks/tools/_index.md

```diff
@@ -23,9 +23,7 @@ Find your preferred operating system below.
 
 ## kind
 
-[`kind`](https://kind.sigs.k8s.io/) lets you run Kubernetes on
-your local computer. This tool requires that you have
-[Docker](https://docs.docker.com/get-docker/) installed and configured.
+[`kind`](https://kind.sigs.k8s.io/) short for Kubernetes in Docker, and as the name suggests, it creates a Kubernetes cluster using Docker to host the nodes. This tool requires that you have [Docker](https://docs.docker.com/get-docker/) installed and configured.
```

Comment:
You don't need Docker to run kind, though you do need a supported local runtime.

---

I don't know that it's helpful to expand the abbreviation. Reason being that people might think you need Docker (you don't), and we're not saying otherwise.

Reply:
BTW https://kind.sigs.k8s.io/docs/design/principles/#target-cri-functionality is out of date.

Reply:
Yeah that's right it don't need supported local runtime but when I read the docs both **kind** and **minikube** definition seemed so similar.  That's why I wanted to point out `Kubernetes IN Docker`.

Reply:
If we're trying to make a distinction, I'd assume that we go beyond stating the Docker runtime. WDYT of rephrasing the entire section till here as the below,

```suggestion
[`kind`](https://kind.sigs.k8s.io/) lets you run Kubernetes on
your local computer. This tool requires that you have a supported
container runtime like [Docker](https://docs.docker.com/get-docker/) installed and configured.
It supports highly available multi-node clusters and builds 
Kubernetes release builds from its source.
```

Reply:
You can also write a contrasting statement in the Minikube section that specifies the Minikube tool being able to only spin up a single node locally.

Reply:
Take care: minikube also [offers](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/) multi node clusters.

Reply:
The background is different: minikube was originally made for helping people to learn; kind was originally made for CI/CD testing.

Reply:
My bad! How do we want to go about listing the differences then? Would we want to state more about the background at all so as to differentiate between the two?